src/EnergyPlus/ChillerIndirectAbsorption.cc

```diff
@@ -1559,6 +1564,19 @@ namespace ChillerIndirectAbsorption {
 		LoopSideNum = IndirectAbsorber( ChillNum ).CWLoopSideNum;
 
 		CpFluid = GetSpecificHeatGlycol( PlantLoop( IndirectAbsorber( ChillNum ).CWLoopNum ).FluidName, EvapInletTemp, PlantLoop( IndirectAbsorber( ChillNum ).CWLoopNum ).FluidIndex, RoutineName );
+		
+		//If there is a fault of Chiller SWT Sensor (zrp_Jun2016)
+		if( IndirectAbsorber( ChillNum ).FaultyChillerSWTFlag && ( ! WarmupFlag ) && ( ! DoingSizing ) && DoWeathSim ){
```

Comment:
This use of DataGlobals::DoWeathSim in many places seems odd.  It isn't clear if you intend for the faults models to apply if the user sets Run Simulation for Sizing Periods to Yes.  (This is not for sizing, but for running the design days with the full model.)  With what you have here the behavior during sizing periods will change with the setting for Run Simulation for Weather File Run Periods.   Flags like DoWeathSim don't really inform what type of simulation is currently happening, for that it might be better to check DataGlobals::KindOfSim.